ckanext/dia_theme/templates/snippets/search_form.html

```diff
@@ -46,6 +46,7 @@
 
     {% block search_facets %}
       {% if facets %}
+        <a class="show-filters button">Add Filter</a>
```

Comment:
In theory, this should be interpolated using the translation helper I think... like this:
```
{{ _('Add Filter') }}
```
It gives people the ability to override the text in a translation file, or just outputs the value provided if no matching translation is found.

Reply:
I'll update this now.